vic/drivers/image/Makefile

```diff
@@ -75,6 +75,7 @@ INCLUDES = -I ${DRIVERPATH}/include \
 
 # Uncomment to include debugging information
 CFLAGS  =  ${INCLUDES} ${NC_CFLAGS}  -ggdb -O0 -Wall -Wextra -std=c99 \
+					 -fopenmp \
```

Comment:
In RASM, we are able to trigger the -openmp (or equivalent) command in the scripts at build time depending on the mpi/openmp pe usage.  If we are all mpi, we do not trigger the openmp build.  We should be able to duplicate this for vic5.  Or is there an actual requirement in vic5 that the openmp flag has to always be set for it to build correctly even when not threading?